### PR TITLE
Kernel and rtc

### PR DESCRIPTION
**Pull request by georgemoralis**
1. Added sceRtcGetCurrentTick

### CODE REVIEW

File: src/core/libraries/kernel/event_flag/event_flag.cpp [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+    if (strlen(pName) >= 32) {
+        return ORBIS_KERNEL_ERROR_ENAMETOOLONG;
+    }
+
+    bool single = true;
+    bool fifo = true;
+
+    switch (attr) {
+    case 0x10:
```
Comment (psucien) [inline src/core/libraries/kernel/event_flag/event_flag.cpp:32]: ORBIS_KERNEL_EVF_ATTR_SINGLE ?

Comment (georgemoralis) [reply src/core/libraries/kernel/event_flag/event_flag.cpp:32]: ? it is createEventFlag not wait..

Comment (georgemoralis) [reply src/core/libraries/kernel/event_flag/event_flag.cpp:32]: yes i will make it more clear :D

**Comment by georgemoralis** [comment]
deprecated

**Pull request closed without merging**